specs/consensus.md

```diff
@@ -53,13 +53,13 @@ Consensus Rules
 
 ## Reserved Namespace IDs
 
-| name                                   | type          | value                                                                |
-| -------------------------------------- | ------------- | -------------------------------------------------------------------- |
-| `TRANSACTION_NAMESPACE_ID`             | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000001` |
-| `INTERMEDIATE_STATE_ROOT_NAMESPACE_ID` | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000002` |
-| `EVIDENCE_NAMESPACE_ID`                | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000003` |
-| `TAIL_PADDING_NAMESPACE_ID`            | `NamespaceID` | `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE` |
-| `PARITY_SHARE_NAMESPACE_ID`            | `NamespaceID` | `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
+| name                                   | type          | value                                                                | description                                                                   |
+| -------------------------------------- | ------------- | -------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| `TRANSACTION_NAMESPACE_ID`             | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000001` | Transactions: requests that modify the state.                                 |
+| `INTERMEDIATE_STATE_ROOT_NAMESPACE_ID` | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000002` | Intermediate state roots, committed after every transaction.                  |
+| `EVIDENCE_NAMESPACE_ID`                | `NamespaceID` | `0x0000000000000000000000000000000000000000000000000000000000000003` | Evidence: fraud proofs or other proof of slashable action.                    |
+| `TAIL_PADDING_NAMESPACE_ID`            | `NamespaceID` | `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE` | Tail padding: padding after all messages to fill up the original data square. |
```

Comment:
Because of the 32 bytes this doesn't look nice when rendered. I wonder if we should write the namespaces as 1, 2, 3, 2^256-2, 2^256-1 respectively and mention that they are represented as 32 bytes.

Reply:
Hmmm. It people view it on https://docs.lazyledger.io/lazyledger-specs then we can configure the CSS however we want, so this might be a non-issue?

Reply:
Yes, let's see how it looks there 👍 In general 1, 2, 3, 2^256-2, 2^256-1 might be easier for a human to parse and can still be used to init a 32 byte array correctly when implementing. Let's keep it as is for now.